### PR TITLE
fixes code example, revises definition, adds &gt;

### PR DESCRIPTION
**Pull request by ericanaughton**
:tea:

ready to go

### CODE REVIEW

File: elements/link.html [hunk 1]
```diff
@@ -25,14 +25,14 @@
...
   </div>
   <pre class="code-example"><code>
-      &lt;html>
-        &lt;head>
-          <strong>&lt;title></strong>Document title<strong>&lt;/title></strong>
-        &lt;/head>
-      &lt;/html>
+      &lt;html&gt;
+        &lt;head&gt;
+          <strong>&lt;link</strong> rel="stylesheet" href="style.css">
```
Comment (whazzmaster) [inline elements/link.html]: Missing an `&gt;` after the word 'link'?

Comment (jremsik-gpsw) [inline elements/link.html]: No, and this is a style issue. The `&gt;` is the `>` at the end of the line.

Comment (ericanaughton) [inline elements/link.html]: what's the style issue?
and yes  it should be      <strong>&lt;link</strong> rel="stylesheet"
href="style.css"&gt;

[image: photo]
_Erica Naughton_
Designer/Developer, Adorable.io
+1 866 440 4593 | http://adorable.io | 123 E. Main St - 2nd Floor, Madison
WI 53715
http://twitter.com/adorableio  http://facebook.com/adorableio
✔ Get a Signature like this: Click Here!
http://ws-promos.appspot.com/r?rdata=eyJydXJsIjogImh0dHA6Ly93d3cud2lzZXN0YW1wLmNvbS9lbWFpbC1pbnN0YWxsP3dzX25jaWQ9NjcyMjk0MTIwJnV0bV9zb3VyY2U9ZXh0ZW5zaW9uJnV0bV9tZWRpdW09ZW1haWwmdXRtX2NhbXBhaWduPXByb21vXzU3MTM1NzMyNTA1OTY4NjQiLCAiZSI6ICI1NzEzNTczMjUwNTk2ODY0In0=

Comment (whazzmaster) [inline elements/link.html]: Ahhh, I see what you're going for now. The `>` at the end of line should be a `&gt;`, as Jim mentioned.

**Comment by whazzmaster** [comment]
My broad feedback is that I like the style of your commit messages; I'd just suggest to capitalize the first word. E.g., "Adds a forgotten &gt; at the end of the code example"

Other than that (which is just going forward- I don't expect you to change existing commits) I'm :+1:

**Comment by ericanaughton** [comment]
Ok thanks, I'll start doing that :)

**Comment by nwerbeckes-gpsw** [comment]
:+1:

**Comment by ericanaughton** [comment]
@jremsik-gpsw does this count as two thumbs up? i added some last minute fixes after zachery's thumbs up.

**Comment by jremsik-gpsw** [comment]
@ericanaughton Yes. Here's a third: :+1:

**Pull request merged**